src/components/common/PickerScreen/styles.js

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+import { whiteColor } from '../../../constants/styleConstants';
+
+const styles = StyleSheet.create({
+  container: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  label: {
+    textAlign: 'center',
+  },
+  section: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: whiteColor,
+    height: 37,
+    width: 230,
```

Comment:
Don't know what case is this one, but try not to use fixed width and height, use paddings instead